inc/saihash.h

```diff
@@ -102,6 +102,8 @@ typedef enum _sai_native_hash_field_t
     /** Native hash field inner destination MAC */
     SAI_NATIVE_HASH_FIELD_INNER_DST_MAC = 17,
 
+    /** Native hash field MPLS labels */
+    SAI_NATIVE_HASH_FIELD_MPLS_LABEL_STACK = 18,
```

Comment:
If we'll look at the rest of this enum's attributes, they all indicate a particular field, with the differentiation between inner and outer. I think we should follow the same pattern here - expand the stack to `SAI_NATIVE_HASH_FIELD_MPLS_LABEL_0`, `SAI_NATIVE_HASH_FIELD_MPLS_LABEL_1` and so on for a fine-grain control.

Then the application can query how many labels ASIC supports by calling sai_query_attribute_enum_values_capability and fill the hash attribute accordingly.

Reply:
makes sense, i can provide up-to five labels to begin with, from _0 to _4.

Reply:
Applied comments